ads/how-to-guides/scheduling-breaks.md

```diff
@@ -45,11 +45,7 @@ An example snippet can be found below:
 
 ### CUE-OUT (Manifest) {#cue-out}
 
-Inside the manifest the ad break start can be signaled by adding an `#EXT-X-CUE-OUT` tag.
-The duration of the ad break should be specified after it.
-The end of the ad break and start of content would be signaled by an `#EXT-X-CUE-IN` tag.
-The `CUE-OUT` tag can be accompanied by a [daterange](#daterange).
-This is interesting if the ad break start does not correspond with the start of a new segment.
+Inside the manifest the ad break start can be signaled by adding an `#EXT-X-CUE-OUT` tag. The duration of the ad break should be specified after it. The end of the ad break and start of content would be signaled by an `#EXT-X-CUE-IN` tag. The `CUE-OUT` tag can be accompanied by a [daterange](#daterange). This is interesting if the ad break start does not correspond with the start of a new segment.
```

Comment:
suggestion: @FabienLavocat I know this wasn't part of the changes you made, but the "This is interesting" maybe should be "This is relevant" or "This is important".